R/add_activity_after_death_flag.R

```diff
@@ -0,0 +1,177 @@
+#' Match on BOXI NRS death dates to process activity after death flag
+#'
+#' @description Match on CHI number where available in the episode file, and add date of death from the BOXI NRS lookup.
+#' Create new activity after death flag
+#'
+#' @param data episode files
+#' @param year financial year, e.g. '1920'
+#' @param deaths_data The death data for the year
+#'
+#' @return data flagged if activity after death
+add_activity_after_death_flag <- function(
+    data,
+    year,
+    deaths_data = read_file(all_slf_deaths_lookup_path())) {
+  # Match on BOXI NRS deaths lookup for records without chi
+  data <- data %>%
```

Comment:
I think this should be named something else. the join at the end isn't working for me

Reply:
maybe something like 
death_activity_lookup <- data %>%
    dplyr::select(year, anon_chi, record_keydate1, record_keydate2,death_date, deceased) %>%....